thepracticaldev/dev.to
# Remove inline style from KatexTag template

## What type of PR is this? (check all applicable)


## Description
Extracting new Katex style into its own CSS file. Also removed redundant import.
## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/pull/8818
## QA Instructions, Screenshots, Recordings
1. Boot up locally and navigate to `/new`
2. Create new article with the following body
```
{% katex %}
\begin{aligned}
(0 * 16 ^5) + (0 * 16 ^4) + (0 * 16 ^3) + (5 * 16 ^2) + (0 * 16 ^1) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) + (0 * 16 ^0) 
\end{aligned} 
{% endkatex %}
```
3. Click preview or save
4. You should be able to scroll the Katex content.
_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?
n/a
